nntrainer/tensor/blas_interface.cpp

```diff
@@ -302,9 +321,14 @@ void scopy(const unsigned int N, const _FP16 *X, const int incX, _FP16 *Y,
   scopy_FP16(N, X, incX, Y, incY);
 }
 
-void scopy(const unsigned int N, const uint8_t *X, const int incX, _FP16 *Y,
-           const int incY) {
-  scopy_INT4(N, X, incX, Y, incY);
+void scopy_int4_to_float16(const unsigned int N, const uint8_t *X,
+                           const int incX, _FP16 *Y, const int incY) {
+  scopy_int4_to_fp16(N, X, incX, Y, incY);
```

Comment:
any reason to use `scopy_int4_to_fp16` instead of having its implementation?

Reply:
I followed convention in blas_interface.cpp
I think it is because we can add-on cuBLAS or any other calculation-accelerating methods

Reply:
makes sense